src/primitives.coffee

```diff
@@ -85,6 +85,19 @@ class Collection extends ElementType
 
   add: (val) -> @push val
 
+  findElements: (cond) ->
+    elements = @content.reduce (results, el) ->
+      return results.concat([el.findElements(cond)]) if el.elementType() in ['array', 'object']
+      return results.concat([el]) if cond(el)
+      return results
+    , []
+    _.flatten elements, true
+
+  find: (cond) ->
```

Comment:
I am still wondering the value then of pulling out the raw values this way. What is the application of this IYO @smizell